features/support/step_definitions/5070230_requesting_additional_hiseq_sequencing_steps.rb

```diff
@@ -1,24 +1,21 @@
 # frozen_string_literal: true
```

Comment:
I had to add the library type association with the request type in here, so just tidied up a bit. Modern Cucumber doesn't rely quite so heavily on regexs.